### ISSUE TITLE
Install fails if python is not installed

### ISSUE THREAD

**Issue by mskri** [webpro/knip#209]
After updating to `knip@2.19.6` when trying to install knip in an environment (i.e. docker) where you don't have python installed you get an error. The readme doesn't seem to indicate that having python installed is a requirement. Not sure if this is a bug report or a request to update the readme? 

Log example

```
 .../node_modules/tree-sitter-bash install$ prebuild-install || node-gyp rebuild
 .../node_modules/tree-sitter install$ prebuild-install || node-gyp rebuild
 .../node_modules/tree-sitter-bash install: prebuild-install warn install No prebuilt binaries found (target=18.17.1 runtime=node arch=x64 libc= platform=linux)
 .../node_modules/tree-sitter-bash install: gyp info it worked if it ends with ok
 .../node_modules/tree-sitter-bash install: gyp info using node-gyp@9.3.1
 .../node_modules/tree-sitter-bash install: gyp info using node@18.17.1 | linux | x64
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python Python is not set from command line or npm configuration
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python Python is not set from environment variable PYTHON
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python checking if "python3" can be used
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python - "python3" is not in PATH or produced an error
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python checking if "python" can be used
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python - "python" is not in PATH or produced an error
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python **********************************************************
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python You need to install the latest version of Python.
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python you can try one of the following options:
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python   (accepted by both node-gyp and npm)
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python - Set the environment variable PYTHON
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python - Set the npm configuration variable python:
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python For more information consult the documentation at:
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python **********************************************************
 .../node_modules/tree-sitter-bash install: gyp ERR! find Python
 .../node_modules/tree-sitter-bash install: gyp ERR! configure error
 .../node_modules/tree-sitter-bash install: gyp ERR! stack Error: Could not find any Python installation to use
 ```

**Comment by webpro** [comment]
That's a bit of a surprise. I've replaced bash-parser with [tree-sitter-node](https://github.com/tree-sitter/node-tree-sitter) + [tree-sitter-bash](https://github.com/tree-sitter/tree-sitter-bash) in v2.19.6. Didn't notice this in their docs and this wasn't clear to me when adding it to Knip.

I've opened https://github.com/tree-sitter/tree-sitter-bash/issues/188 over there, maybe it's just an overlook with their v0.20.0 release (I could downgrade to v0.19.0 if necessary).
